composer.json

```diff
@@ -49,7 +50,7 @@
         "symfony/browser-kit": "^2.8 || ^3.0",
         "symfony/dom-crawler": "^2.8 || ^3.0",
         "polishsymfonycommunity/symfony-mocker-container": "^1.0",
-        "matthiasnoback/symfony-dependency-injection-test": "^1.0",
+        "matthiasnoback/symfony-dependency-injection-test": "^1.0 || ^2.0",
```

Comment:
do we need both versions to support the range of php versions we support?

Reply:
Yes, we can't install PHPUnit 6.0 because of a conflict on `sebastian/exporter`.